ocs_ci/ocs/bucket_utils.py

```diff
@@ -3006,3 +3014,71 @@ def get_obj_versions(mcg_obj, awscli_pod, bucket_name, obj_key):
             d["ETag"] = d["ETag"].strip('"')
 
     return versions_dicts
+
+
+def compare_object_versions(mcg_obj, awscli_pod, bucket_a, bucket_b, obj_key):
+    """
+    Compare the versions of an object in two different buckets
+
+    Args:
+        mcg_obj (MCG): MCG object
+        awscli_pod (Pod): Pod object where AWS CLI is installed
+        bucket_a (str): Name of the first bucket
+        bucket_b (str): Name of the second bucket
+        obj_key (str): S3 path to the object in the bucket
+
+    Returns:
+        bool: True if the versions match, False otherwise
+    """
+    logger.info(f"Comparing the versions of {obj_key} in {bucket_a} and {bucket_b}")
+
+    source_versions = get_obj_versions(mcg_obj, awscli_pod, bucket_a, obj_key)
+    source_etags = [v["ETag"] for v in source_versions]
+    target_versions = get_obj_versions(mcg_obj, awscli_pod, bucket_b, obj_key)
+    target_etags = [v["ETag"] for v in target_versions]
+    if source_etags != target_etags:
+        logger.warning(
+            (
+                f"The versions of {obj_key} in {bucket_a} and {bucket_b} do not match:"
+                f"{source_etags} != {target_etags}"
+            )
+        )
+        return False
+    logger.info(f"The versions of {obj_key} in {bucket_a} and {bucket_b} match")
+    return True
+
+
+def wait_for_object_versions_match(
+    mcg_obj, awscli_pod, bucket_a, bucket_b, obj_key, timeout=600
```

Comment:
Same for bucket name here as well

Reply:
Ack, I'll replace `bucket_a` with `first_bucket` and `bucket_b` with `second_bucket` to make the methods more readable.

Reply:
Fixed in https://github.com/red-hat-storage/ocs-ci/pull/11438/commits/0eda7a886d95a7d462b578683e127fc71500d443